rubric/rubric_quality.md

```diff
@@ -15,6 +15,6 @@
     <td>Code is generally easy to read and understand with few non-reoccurring issues and at most two reoccurring issue that affects readability. <br></br> Multiple issues with code repetition exist, and several tests are absent and/or are of poor efficacy</td>
```

Comment:
I feel like something like the following might be a more accurate description of an "**Unsatisfactory**" quality of code:

_Code is fairly readable but not very easy to understand. Sufficient and proper commenting/documentation is missing. A few non-reoccurring, and at most two reoccurring issues exist that affect readability._